kong/templates/nginx_kong.lua

```diff
@@ -320,4 +323,25 @@ server {
     }
 }
 > end
+
+> if role == "control_plane" then
+server {
```

Comment:
I think it is missing e.g. `real_ip` as mentioned in other comment.

Reply:
See my above comment, it won't work because of mTLS requirements here.